glue_wwt/viewer/jupyter/tests/test_jupyter_viewer.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import pytest
+
+from glue_jupyter.app import JupyterApplication
+
+from ..jupyter_viewer import WWTJupyterViewer
+
+from ...tests.test_base import BaseTestWWTDataViewer
+
+DATA = os.path.join(os.path.dirname(__file__), 'data')
+
+
+# class WWTJupyterViewerBlocking(WWTJupyterViewer):
+
+#     def _initialize_wwt(self):
+#         from pywwt.jupyter import WWTJupyterWidget
+#         self._wwt = WWTJupyterWidget(block_until_ready=sys.platform != 'win32')
+
+
+@pytest.mark.xfail
```

Comment:
@Carifio24 - somewhat important, though of course the Jupyter widget wasn't tested on main. It seems we have to actually launch a Jupyter instance for pywwt to work correctly, will try and think about how we can go about doing this, I'll try and check how we actually test pywwt in Jupyter.